SitumWayfinding/Classes/Positioning/Internal/MarkerRenderer.swift

```diff
@@ -22,87 +22,114 @@ class MarkerRenderer {
     }
     
     func displayPOIMarkers(forFloor floor: SITFloor) {
-        cleanMarkers()
+        removeMarkers()
         let poisInFloor = buildingInfo.indoorPois.filter { poi in poi.position().floorIdentifier == floor.identifier }
     
         for poi in poisInFloor {
             let marker = SitumMarker(poi)
             markers.append(marker)
-            loadUnselectedIcon(forMarker: marker)
+            loadIcon(forMarker: marker, selected: false) { [weak self] marker in
+                self?.insertMarkerInGoogleMaps(marker: marker)
+            }
```

Comment:
Descaoplei a carga do icono e a asignacion a google maps. Creo que está máis claro así que é o que se fai.